helpers/bot_client.py

```diff
@@ -290,6 +290,22 @@ async def flip(self, ctx, number=1):
         except Exception as e:
             await ctx.send(str(e))
 
+    @commands.command(name='roll', help='Rolls a dice of your choosing')
+    async def roll(self, ctx, number=1, dice_size="D6"):
+        total=0
+        try:
+            await ctx.send(f'Rolling a {dice_size} {number}time(s)...')
+            for i in range(number):
+                result = random.randint(1, int(dice_size.lower().replace("d","")))
```

Comment:
nit - change the double quotes to single quotes in the `replace()` method to match the rest of the code, also add a space after the comma between the arguments in `replace()`, i.e. `.replace('d', '')`